packages/react/src/components/progress-tracker/progress-tracker.tsx

```diff
@@ -173,20 +177,20 @@ const Step: VoidFunctionComponent<StepProps> = ({
 
     if (stepNumber === value) {
         dataTestId = 'progress-tracker-step-current';
-        StepComponent = CurrentStep;
+        StepComponent = ActiveStep;
     } else if ((linear && stepNumber < value) || (!linear && step.completion === 'completed')) {
         dataTestId = 'progress-tracker-step-completed';
         screenReaderText = t('completedAriaLabel');
-        StepComponent = CompletedStep;
+        StepComponent = CompleteStep;
     } else {
         dataTestId = 'progress-tracker-step-uncompleted';
         screenReaderText = t('uncompletedAriaLabel');
-        StepComponent = UncompletedStep;
+        StepComponent = IncompleteStep;
```

Comment:
Tu as changé `Uncompleted` pour `Incompleted`. 
Selon le tableau plus bas, je crois que Uncompleted est plus approprié dans ce cas-ci.
Uncompleted | Incompleted
-- | --
Describes something not finished or accomplished | Usually used in grammar or linguistics
Refers to tasks, projects, physical objects | Refers to verbs or sentences missing required elements

J'ai été laisser un commentaire dans Figma pour faire la modification

Reply:
Dans le même ordre idée, j'ai changé `Complete` pour `Completed`. 
Si c'est bon, on pourrait l'ajouter à la note sur Figma.

Reply:
la modification est fait dans Figma.
I thank you!